esp/esp/program/controllers/lunch_constraints.py

```diff
@@ -127,7 +127,12 @@ def get_lunch_subject(self, day):
             new_subject.category = category
             new_subject.class_info = 'Enjoy a break for lunch with your friends!  Please register for at least one lunch period on each day of the program.'
             new_subject.class_size_min = 0
-            new_subject.class_size_max = self.program.program_size_max
+            # If the program doesn't have a max size, we unfortunately still
+            # need one here.  Set a really big one.
+            if self.program.program_size_max:
+                new_subject.class_size_max = self.program.program_size_max
+            else:
+                new_subject.class_size_max = 10**6
```

Comment:
This can be one line: ``new_subject.class_size_max = self.program.program_size_max or 10**6``.